scientific_library/tvb/datatypes/time_series.py

```diff
@@ -82,6 +83,8 @@ def nr_dimensions(self):
 
     @property
     def sample_rate(self):
+        if len(self.sample_period_unit) > 0 and self.sample_period_unit[0] == "m":
```

Comment:
`m` is *not* a unit of time.  I can't access the HBP issues on Jira but this is a red flag to me.

Reply:
I just gave you access on HBP project.
Nevertheless, I agree in red flagging this. This is still unfinished work.